feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -186,12 +146,11 @@ func TestHardwarecards(t *testing.T) {
 			fwVerValidation:       false,
 			rrValidation:          false,
 			operStatus:            activeStatus,
-			parent:                "",
-			pType:                 componentType["fan"],
+			parent:                oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CHASSIS,
```

Comment:
these are vendor dependent values and rather than this we should check for eventual parent that should be chasis. We did the same in telemetry basic check